OpenTabletDriver.UX/Windows/Configurations/ConfigurationEditor.cs

```diff
@@ -153,9 +153,14 @@ private void WriteConfigurations(IEnumerable<TabletConfiguration> configs, Direc
 
                 var path = Path.Join(dir.FullName, manufacturer, string.Format("{0}.json", tabletName));
                 var file = new FileInfo(path);
-                if (!file.Directory.Exists)
-                    file.Directory.Create();
-                Serialization.Serialize(file, config);
+                try
+                {
+                    if (!file.Directory.Exists)
+                        file.Directory.Create();
+                    Serialization.Serialize(file, config);
+                } catch (System.UnauthorizedAccessException) {
```

Comment:
Might have to add a using statement here to remove the namespace from the class name
```suggestion
                }
                catch (UnauthorizedAccessException)
                {
```